src/click/termui.py

```diff
@@ -204,7 +205,7 @@ def confirm(
         text,
         prompt_suffix,
         show_default,
-        "y/n" if default is None else ("Y/n" if default else "y/N"),
+        _("y/n") if default is None else (_("Y/n") if default else _("y/N")),
```

Comment:
Don't translate these, they're related to the literal characters that need to be input.